### PR TITLE
Tiny fix for rpm and deb descriptions

### PR DESCRIPTION
**Pull request by hqhq**
Signed-off-by: Qiang Huang h.huangqiang@huawei.com

### CODE REVIEW

File: hack/make/.build-deb/control [hunk 1]
```diff
@@ -15,9 +15,9 @@ Recommends: aufs-tools,
             ${apparmor:Recommends}
 Conflicts: docker (<< 1.5~), docker.io, lxc-docker, lxc-docker-virtual-package
 Description: Docker: the open-source application container engine
- Docker is an open source project to pack, ship and run any application as a
- lightweight container
- .
+ Docker is an open source project to build, ship and run any applications as a
```
Comment (cpuguy83) [inline hack/make/.build-deb/control]: This should be `ship and run any application` where `application` is singular.

**Comment by thaJeztah** [comment]
Could you address @cpuguy83's comment in both files? Then LGTM

File: hack/make/.build-deb/control (hunk 1 above)
Comment (hqhq) [inline hack/make/.build-deb/control]: Fixed, thanks.

**Comment by cpuguy83** [comment]
LGTM

File: hack/make/.build-deb/control [hunk 2]
```diff
@@ -15,9 +15,9 @@ Recommends: aufs-tools,
             ${apparmor:Recommends}
 Conflicts: docker (<< 1.5~), docker.io, lxc-docker, lxc-docker-virtual-package
 Description: Docker: the open-source application container engine
- Docker is an open source project to pack, ship and run any application as a
- lightweight container
- .
+ Docker is an open source project to build, ship and run any application as a
+ lightweight container.
+
```
Comment (tianon) [inline hack/make/.build-deb/control]: The period on this blank line is actually important for how this file is parsed; can we get that added back please? :+1:

Comment (hqhq) [inline hack/make/.build-deb/control]: Sorry, you mean added what back?

Comment (thaJeztah) [inline hack/make/.build-deb/control]: IIUC, the period (`.`) must be on a separate line (like it was)

Comment (hqhq) [inline hack/make/.build-deb/control]: Thanks, don't know how we would depend on this kind of format, I reverted this one, if so, do we need to change rpm spec as well? To add the period on the blank line?

Comment (thaJeztah) [inline hack/make/.build-deb/control]: Sorry, I don't know the reason behind this; I just spotted the other file has a blank line as well;

@tianon, could you shed some light, and check the other file as well before I merge?

**Comment by jfrazelle** [comment]
LGTM, ping @tianon

**Comment by thaJeztah** [comment]
there's a question for @tianon in https://github.com/docker/docker/pull/15367#discussion_r36603535

**Comment by tianon** [comment]
LGTM

For more information about the format of the Debian file, see https://www.debian.org/doc/debian-policy/ch-controlfields.html, especially:

> Paragraph separators (empty lines) and lines consisting only of spaces and tabs are not allowed within field values or between fields. Empty lines in field values are usually escaped by representing them by a space followed by a dot.

**Comment by thaJeztah** [comment]
Thanks @tianon

**Pull request merged**